fedimintd/templates/run.html

```diff
@@ -1,7 +1,6 @@
 {% extends "base.html" %}
 
 {% block title %} Fedimint {% endblock %}
-{% block head %} <meta http-equiv="refresh" content="5" /> {% endblock %}
```

Comment:
Doesn't this need to be conditional on `has_connection_string` instead of removing it outright? How will we get to the QR after the DKG has finished? If you tested with 1-of-1 latency might have hidden that problem.

Reply:
I think you're right in the principle, and I didn't fully understand what this code is doing. But it also looks like we actually not displaying this page until dkg finished? @justinmoon

Reply:
In that case we should remove `has_connection_string` and the branch.

Reply:
We should make it show something again while the dkg is running. But that could be in another PR. I've make it conditional now. :)